README.md

```diff
@@ -1,181 +1,196 @@
+# LTI 1.3 Advantage Library
 
-| **Note** : If you are looking for the example tool that uses this library, it has been moved into its own repo https://github.com/IMSGlobal/lti-1-3-php-example-tool |
-| --- |
+A library used for building IMS-certified LTI 1.3 tool providers in PHP.
 
-# LTI 1.3 Advantage Library
-This code consists of a library for creating LTI tool providers in PHP.
+## Installation
 
-# Library Documentation
+Run:
 
-## Importing the library
-### Using Composer
-Add the following to your `composer.json` file
-```json
-"repositories": [
-    {
-        "type": "vcs",
-        "url": "https://github.com/IMSGlobal/lti-1-3-php-library"
-    }
-],
-"require": {
-    "imsglobal/lti-1p3-tool": "dev-master"
-}
+```bash
+composer require packbackbooks/lti-1p3-tool
 ```
-Run `composer install` or `composer update`
-In your code, you will now be able to use classes in the `\IMSGlobal\LTI` namespace to access the library.
 
-### Manually
-To import the library, copy the `lti` folder inside `src` into your project and use the following code at the beginning of execution:
+In your code, you will now be able to use classes in the `Packback\Lti1p3` namespace to access the library.
+
+### Configuring JWT
+
+Add the following when bootstrapping your app.
+
 ```php
-require_once('lti/lti.php');
-use \IMSGlobal\LTI;
+use Firebase\JWT\JWT;
+
+JWT::$leeway = 5;
```

Comment:
As somebody not familiar with `Firebase` or the `$leeway` field, I have no clue what this bootstrapping is doing.

Reply:
That makes two of use :laughing: